client/src/components/CompanyRegister.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from "react";
+import {
+  Form,
+  Input,
+  Dropdown,
+  Button,
+  TextArea,
+  Message,
+  Grid,
+  Header
+} from "semantic-ui-react";
+
+import { companyRegister } from "../api/companies";
+import { industryList } from "../utils/constants";
+
+class CompanyRegister extends Component {
+  state = {
+    companyProfile: {
+      name: "",
+      description: "",
+      industry: [],
+      email: "",
+      password: "",
+      user_id: ""
+    },
+    error: null,
+    success: null
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+
+    companyRegister(this.state.companyProfile)
+      .then(data => {
+        console.log(data);
+        if (data.success) {
+          this.setState({
+            companyProfile: {
+              name: "",
+              description: "",
+              industry: [],
+              email: "",
+              password: ""
+            },
+            success: true
+          });
+        } else {
+          this.setState({});
```

Comment:
this means that if there `data.success` is `false`, then reset the form? Is that the behaviour you're after?

Reply:
data.success should be true to reset the form, if it's false then it will return am error message .